### PR TITLE
V1 Breadcrumbs to V3

### PR DESCRIPTION
**Pull request by Andrew565**
## Summary

- Replacing V1 breadcrumbs components with their V3 equivalents
- This is being done as part of DST's work to remove all remaining V1 component instances.
- I work for the DST

## Related issue(s)

- Closes DST [#2658](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2658)

## Testing done

- Tested locally.

## Screenshots

N/A

## What areas of the site does it impact?

Affects several parts of the site where V1 breadcrumbs still exist.

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [X] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [X] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [X] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [X] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [X] The vets-website header does not contain any web-components
- [X] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [X] I reached out in the `#sitewide-public-websites` Slack channel for questions

### CODE REVIEW

**Review by williamphelps13 — DISMISSED** [review]
The changes to the `src/applications/accredited-representative-portal/components/common/Breadcrumbs.jsx` file and the corresponding test changes `src/applications/accredited-representative-portal/tests/components/common/Breadcrumbs.unit.spec.js` look good to me.

**Review by powellkerry — DISMISSED** [review]

**Review by acrollet — DISMISSED** [review]
approved for cto health changes

**Review by micahchiang — APPROVED** [review]

**Review by acrollet — APPROVED** [review]

**Review by williamphelps13 — APPROVED** [review]

**Review by rhasselle-oddball — APPROVED** [review]

File: src/applications/simple-forms/21-4138/components/breadcrumbs.jsx [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+    },
+    {
+      href:
+        '/supporting-forms-for-claims/statement-to-support-claim-form-21-4138',
+      label: 'Submit a statement to support a claim',
+    },
+  ];
+  const bcString = JSON.stringify(breadcrumbs);
+  return (
+    <va-breadcrumbs
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-4138/components/breadcrumbs.jsx:18]: Just for curiosity - Is the old approach still OK as well? I assume `VaBreadcrumbs` are also on v3 now? or is `va-breadcrumbs` doing something special here that `VaBreadcrumbs` can't handle? Just thinking, as a consumer, if I had the preference I would prefer not to stringify my params unless necessary.

On another note, I just introduced some new code using `VaBreadcrumbs` at `simple-forms/mock-form-minimal-header/components/breadcrumbs.jsx` today. I can adjust that as a followup if this is the new recommendation.

Comment (Andrew565) [reply src/applications/simple-forms/21-4138/components/breadcrumbs.jsx:18]: I believe using the react-binding version - VaBreadcrumbs - is still supported. I was just converting everything to use the web component version - va-breadcrumbs - and standardized on that.

**Pull request merged**